resources/views/back/fragments/_partials/form.blade.php

```diff
@@ -11,6 +11,8 @@
         ]) !!}
         @endif
 
+        {!! Form::media($fragment, 'images', 'image') !!}
```

Comment:
I think I'd rather show the user either a text input or an image uploader, not both. Maybe we could dump the `html` flag and replace it by `type`?